internal/util/testutil/pg.go

```diff
@@ -129,13 +128,13 @@ func Table(ctx context.Context, tb testing.TB, pool *pgdb.Pool, db string) strin
 	table := TableName(tb)
```

Comment:
@rumyantseva Please clean-up those helpers in your PR. For example, it is not clean if this helper return table name (with a hash) or collection name. At very least, documentation should be updated. Maybe name too

Reply:
👌